pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 100
+target-version = ['py37']
```

Comment:
We needed to move this out of setup.cfg due to issues with the config for pydocstyle. Read more [here](https://github.com/psf/black/issues/900)